src/pieces/commands/onboarding.py

```diff
@@ -0,0 +1,191 @@
+        
+        user_input = input(get_prompt()).strip()
+        while user_input != self.predicted_text:
+            if user_input == "exit":
+                sys.exit(1)
+            console.print(Markdown(f"❌ Wrong command entered, You should type: `{self.predicted_text}`"))
+            user_input = input(get_prompt()).strip()
+
+        run_command(*extract_text(user_input.removeprefix("pieces ")))
+
+        return True
+
+def create_snippet_one_validation():
+    text = pyperclip.paste()
+    normalized_s1 = '\n'.join(line.strip() for line in text.strip().splitlines())
+    normalized_s2 = '\n'.join(line.strip() for line in demo_snippet.strip().splitlines())
+
+    return normalized_s1 == normalized_s2 , "Seems you did not copy the snippet yet"
+
+def onboarding_command(**kwargs):
+    console = Console()
+    step_number = 1
+    steps = {
+        "Step 1: Creating a snippet":[
+            OnboardingStep("Let's get started by saving a snippet\n"
+                    "copy the following snippet python snippet: \n"
+                    f"```python\n{demo_snippet}\n```",
+                    create_snippet_one_validation
+                ),
+            OnboardingCommandStep(
+                "You can save a snippet by typing `pieces create`",
+                "pieces create"
+            )
+        ],
+        "Step 2: Opening the created snippet":[
+            OnboardingCommandStep(
+                "Now let's checkout all the saved snippets by typing `pieces list`",
+                "pieces list"
+            )
+        ],
+        "Step 3: Run in loop":[
+            OnboardingStep(
+                "Running in loop help you to create a session with the Copilot which we will be convering " 
+                "or perform an opration on the same snippet multiple times which will increase your **productivity**",
+                lambda: (True, "")
+            ),
+            OnboardingCommandStep(
+                "You can run in loop by typing `pieces run` Don't forget to exit the loop by typing `exit`",
+                "pieces run"
+            )
+        ],
+        "Step 5: Chat with the Copilot":[
+            OnboardingCommandStep(
+                "Now you understood what is `pieces run` let's try to interact with the Copilot by typing `pieces ask 'How to print I love Pieces CLI in Python and Java'`",
+                "pieces ask 'How to print I love Pieces CLI in Python and Java'"
+            ),
+            OnboardingCommandStep(
+                "Well you just asked one question in a conversation, " "Let's create a session with Copilot by typing `pieces run` then use `ask` to interact with Copilot",
+                "pieces run"
+            ),
+            
+        ],
+        "Step 6: Sharing your feedback" : [
+            OnboardingCommandStep(
+                "Your feedback is very **important** for us to improve feel free to share your feedback by typing `pieces feedback`",
```

Comment:
"Your feedback is very **important** to us. Please share some of your feedback by typing `pieces feedback`."